package.json

```diff
@@ -2,18 +2,41 @@
   "name": "node-crypto",
   "version": "0.0.1",
   "description": "Easy (yet strong) encryption and decryption facilities for Node.js",
-  "main": "index.js",
+  "main": "lib/crypto.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "clean": "rm -rf lib coverage* .nyc_output",
+    "build": "npm run clean && babel src --out-dir lib",
+    "coverage": "nyc report --reporter=text-lcov > coverage.lcov && codecov",
+    "test": "retire -n -p package.json && npm run build && npm run unit",
+    "unit": "nyc --require babel-core/register mocha test/src/**"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/elastic/node-crypto.git"
   },
-  "author": "",
+  "author": "Shaunak Kashyap (https://github.com/ycombinator)",
   "license": "Apache-2.0",
   "bugs": {
     "url": "https://github.com/elastic/node-crypto/issues"
   },
-  "homepage": "https://github.com/elastic/node-crypto#readme"
+  "homepage": "https://github.com/elastic/node-crypto#readme",
+  "devDependencies": {
+    "@elastic/eslint-config-kibana": "^0.2.0",
+    "babel-cli": "^6.14.0",
+    "babel-core": "^6.14.0",
+    "babel-eslint": "^6.1.2",
+    "babel-plugin-transform-async-functions": "^6.8.0",
+    "babel-plugin-transform-regenerator": "^6.14.0",
+    "babel-plugin-transform-runtime": "^6.15.0",
+    "babel-polyfill": "^6.13.0",
+    "babel-preset-es2015": "^6.14.0",
+    "eslint": "^3.4.0",
+    "eslint-plugin-mocha": "^4.5.1",
+    "expect.js": "^0.3.1",
+    "mocha": "^3.0.2",
+    "nyc": "^8.1.0",
+    "regenerator": "^0.8.46",
+    "retire": "^1.2.10",
+    "sinon": "^1.17.5"
```

Comment:
I think we should lock our versions here (remove the `^`) since humans screw up semver all the time.